python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -461,35 +528,68 @@ def fa_forces(row: pd.Series) -> pd.Series:
         self.mirror = self.interpolated.merge(
             applied, how="left", left_index=True, right_index=True
         )
+        self.mirror.dropna(inplace=True)
+        logging.info(f"Processing {len(self.mirror.index)} records")
 
     def plot_acc_forces(self) -> None:
```

Comment:
Some inline comments in this function would be useful for the reader to see faster what each plot is

Reply:
The methods come with one-line documentation. I think that's enough to explain what is being plotted. Can you be more specific about what's missing?

Reply:
I think it's fine looking at it again. Thanks!